etc/rc.conf

```diff
@@ -0,0 +1,12 @@
+###############################################################################
+# rc.conf is loaded by /lib/init/rc.boot and /lib/init/rc.shutdown as early
+# as possible. Please inspect these scripts for more details.
+#
+# Essentially, rc.conf is for config variables, and /etc/rc.d/ is for scripts.
+# Boot scripts should be suffixed with .boot
+# Shutdown scripts should be suffixed with .pre.shutdown or .post.shutdown
+#
+# Example: Create a dmesg save script.
+#
+# -> echo "dmesg > /var/log/dmesg.log" > /etc/rc.d/dmesg.boot
+###############################################################################
```

Comment:
I think it's clearer to remove this message as users might add this example to `rc.conf`. It's better if we instead replace this with a link to a Wiki page full of example additions to `/etc/rc.d`.